tests/phpunit/tests/media/shortcodes/test-wpml-page-builders-media-shortcodes.php

```diff
@@ -141,6 +141,181 @@ public function it_should_translate() {
 		$this->assertEquals( $translated_content, $actual_content );
 	}
 
+	/**
+	 * @test
+	 */
+	public function it_should_return_false_when_content_has_no_shortcode() {
+		$content = 'Some content with no shortcode';
+
+		$media_translate = $this->get_media_translate();
+
+		$config = array();
+
+		$subject = $this->get_subject( $media_translate, $config );
+
+		$this->assertFalse( $subject->has_media_shortcode( $content ) );
+	}
+
+	/**
+	 * @test
+	 * @group wpmlpb-166
+	 */
+	public function it_should_return_true_when_content_has_media_shortcodes() {
+		$original_url_1 = 'http://example.org/dog.jpg';
+		$original_url_2 = 'http://example.org/england.jpg';
+		$original_id_1  = 2;
+		$original_id_2  = 7;
+
+		$content = '
```

Comment:
@kagg-design, to make the test more readable, you can reduce the number of shortcodes in the config and also reduce the content:

```php
$content = '[et_pb_gallery _builder_version="3.12" gallery_ids="1,2" /]';

$config = array(
     array(
				'tag'        => array( 'name' => 'et_pb_gallery' ),
				'attributes' => array(
					'gallery_ids' => array( 'type' => WPML_Page_Builders_Media_Shortcodes::TYPE_IDS ),
				),
			),
);
```

Reply:
Done